src/pages/About.jsx

```diff
@@ -20,7 +20,11 @@ const developers = [
   {
     name: 'Gonzalo Lara',
     img: testPic,
-    description: 'Todo: about Gonzalo Lara',
+    description: 'My passion for computers has lead me to pursue a career in Software '
+      + 'Engineering. I am currently a UWB CSS student completing my final year in the '
+      + 'Computer Science and Software Engineering program. I am 24 years old, bilingual '
+      + 'and currently a part of the DACA program.  I intend to contribute as much as '
+      + 'possible to building a better tomorrow.',
```

Comment:
Better to leave this in the other PR so that it gets merged quickly. It's also not relevant to the Login Button fix.

Reply:
Ok I did that